drivers/modem/ublox-sara-r4.c

```diff
@@ -1392,9 +1524,9 @@ static void modem_reset(void)
 
 	LOG_INF("Waiting for network");
 
-	/* wait for +CREG: 1 notification (20 seconds max) */
+	/* wait for +CREG: 1 notification (20 seconds max), or 5 for roaming */
```

Comment:
Good call, I too have added this into my stand alone version.

We may wish to make ROAMING option configurable, thoughts?